internal/hcp/registry/types.iterations.go

```diff
@@ -152,3 +116,59 @@ func (i *Iteration) AddLabelsToBuild(buildName string, data map[string]string) e
 	i.StoreBuild(buildName, build)
 	return nil
 }
+
+// AddSHAToBuildLabels adds the Git SHA for the current iteration (if set) as a label for all the builds of the iteration
+func (i *Iteration) AddSHAToBuildLabels(sha string) {
+	i.builds.Range(func(_, v any) bool {
+		b, ok := v.(*Build)
+		if !ok {
+			return true
+		}
+
+		b.MergeLabels(map[string]string{
+			"GIT_SHA": sha,
```

Comment:
I do wonder if this should be a different label name. But we don't have any details on what folks expect to see. So I don't have anything to add.

Reply:
I would make it lower case `git_sha` because all of our generated labels are lower case. 🤔

Reply:
Fair enough, that can be renamed `git_sha` for consistency, thanks Moss